service/history/workflow/mutable_state_impl.go

```diff
@@ -729,6 +735,7 @@ func (e *MutableStateImpl) GetCompletionEvent() (*historypb.HistoryEvent, error)
 		e.executionState.RunId,
 		firstEventID,
 		completionEventID,
+		lastWriteVersion,
```

Comment:
I'm not sure this is the right version to be using here, but it seems to make sense.